### ISSUE TITLE
Horizontal number picker scrolls when disabled

### ISSUE THREAD

**Issue by radixdiaboli** [ShawnLin013/NumberPicker#29]
If I disable the number picker programmatically, it will still scroll in response to touch events.  This doesn't seem to happen when the orientation is vertical, only horizontal.

**Comment by radixdiaboli** [comment]
Issue goes away if I do the following:

```
public class NumberPicker extends com.shawnlin.numberpicker.NumberPicker {
    public NumberPicker(Context context) {
        super(context);
    }

    public NumberPicker(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public NumberPicker(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent event) {
        return isEnabled() && super.onInterceptTouchEvent(event);
    }
}

```

**Comment by ShawnLin013** [comment]
🐛 Fixed at v2.4.0.

**Comment by varunchandran333** [comment]
I am using version 2.4.12, I got the issue 

https://user-images.githubusercontent.com/24404413/108030982-1498f900-7056-11eb-93ca-722174074361.mp4
